certificate-service/pdfbox-generator/src/main/java/se/inera/intyg/certificateservice/pdfboxgenerator/pdf/value/PdfValueGeneratorUtil.java

```diff
@@ -11,9 +11,13 @@ private PdfValueGeneratorUtil() {
     throw new IllegalStateException("Utility class");
   }
 
-  public static List<String> splitByLimit(Integer limit, String s) {
+  public static List<String> splitByLimit(Integer limit, String s,
+      boolean addOverflowOnContinuationSheet) {
```

Comment:
Instead of adding this logic to each of the values, isn't it better than in the diagnosis question for 3226 and 7809 we don't submit an overflow sheet, and the logic if added on an overflow sheet will only apply if the sheet is available